app/scripts/modules/core/utils/renderIfFeature.directive.ts

```diff
@@ -0,0 +1,34 @@
+import {module} from 'angular';
+import {DirectiveFactory} from './tsDecorators/directiveFactoryDecorator';
+
+class RenderIfFeatureDirectiveController {
+  public $attrs: ng.IAttributes;
+  public $element: JQuery;
+
+  constructor(private settings: any) {}
+
+  public initialize(): void {
+    let feature: string = this.$attrs['renderIfFeature'];
+    if (feature && !this.settings.feature[feature]) {
+      this.$element.addClass('hidden');
```

Comment:
What happens if you try to remove the element altogether? My only concern is that the element will still be rendered and run through the Angular lifecycle, and that may result in surprising behavior if some things are only enabled by the feature.